modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DXPCloudSetup.testcase

```diff
@@ -27,514 +27,667 @@ definition {
-			accountRoleTitle = "Account Administrator",
-			accountUserName = "Test Test");
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CustomerPortal.enterOnProject(title = "Test Account 01");
+			CustomerPortal.enterOnProject(title = "Test Account LXC-SM");
+		}
 
-		CustomerPortal.clickOnStartProjectSetup();
+		task ("Going through onboardig") {
+			CustomerPortal.clickOnStartProjectSetup();
 
-		CustomerPortal.clickOnSkipForNowButton();
+			CustomerPortal.clickOnSkipForNowButton();
 
-		CustomerPortal.clickOnSkipForNowButton();
+			CustomerPortal.clickOnSkipForNowButton();
+		}
 
-		CP2NavigationMenu.goToProductActivation();
+		task ("Go to the product activation page and open the card activation modal") {
+			CP2NavigationMenu.goToProductActivation();
 
-		CP2NavigationMenu.clickOnProductButton(product = "DXP Cloud");
+			CP2NavigationMenu.clickOnProductButton(product = "LXC-SM");
 
-		CP2DXPCloudSetup.clickOnFinishActivationButton();
+			CustomerPortal.clickOnFinishActivationButton();
+		}
 
-		AssertElementPresent(locator1 = "CP2DXPCloudSetup#SETUP_DXP_CLOUD_PAGE");
+		task ("Assert modal Setup LXC-SM") {
+			AssertElementPresent(
+				locator1 = "CustomerPortalSite#MODAL_PAGE",
+				modal = "Set up LXC-SM");
+		}
 	}
 
-	@description = "LPS-146686 - Verify if the activation status of the card changes to (In Progress) after filling out the DXP Cloud Set up"
+	@description = "Automation ID: LPS-146686 | Test Summary: Verify if the activation status of the card changes to (In Progress) after filling out the DXP Cloud Set up"
 	@priority = "4"
 	test SeeIfTheCardActivationStatusChanges {
-		CP2DXPCloudSetup.createDXPCloudObjects();
-
-		Accounts.openAccountsAdmin();
+		task ("Creating objects for the LXC-SM") {
+			CP2DXPCloudSetup.createDXPCloudObjects();
+		}
 
-		CustomerPortal.accountTableProject(title = "Test Account 01");
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
 
-		Accounts.addAssigneeToRoleViaUsersTab(
-			accountRoleTitle = "Account Administrator",
-			accountUserName = "Test Test");
+			CustomerPortal.accountTableProject(title = "Test Account LXC-SM");
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
 
-		CustomerPortal.enterOnProject(title = "Test Account 01");
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CustomerPortal.clickOnStartProjectSetup();
+			CustomerPortal.enterOnProject(title = "Test Account LXC-SM");
+		}
 
-		CustomerPortal.clickOnSkipForNowButton();
+		task ("Going through onboardig") {
```

Comment:
Luana, please check the typo: "onboarding"

Reply:
Done.